lib/core/src/lib/form/models/form-rules.model.ts

```diff
@@ -42,7 +42,7 @@ export abstract class FormRulesManager<T> {
     constructor(protected formService: FormService) {}
 
     protected formModel: FormModel;
-    private onDestroy$ = new Subject<boolean>();
+    protected onDestroy$ = new Subject<boolean>();
     private initialized = false;
 
     initialize(formModel: FormModel) {
```

Comment:
do we still nedd to pass formModel? It looks like we are not using anymore the method paramater, but only the class field